src/validation_workflow/tar/validation_tar.py

```diff
@@ -72,5 +66,5 @@ def cleanup(self) -> bool:
             self.os_process.terminate()
             self.osd_process.terminate()
         except:
-            raise Exception('Failed to Stop Cluster')
+            raise Exception('Failed to terminate the processes that started OS and OSD')
```

Comment:
We spell out `OpenSearch` and `OpenSearch Dashboards` in other exceptions, and finish sentences with `.`. Can we try to please make errors consistent?

Reply:
Sure @dblock, I'll update all the output messages to stay in format.